### DISCUSSION TITLE
Adding a color accent to blur background

### DISCUSSION THREAD

**Discussion by ip-web** [MicaForEveryone/MicaForEveryone#144] [Ideas]
The blur behind option is great but if the apps behind is white or clear it makes the white text or content usually unreadable, would it be possible to add a color accent or darken the blur to make it usable in any situation ?

**Comment by xmine64** [comment]
Not sure, @dongle-the-gadget @rounk-ctrl ?

**Reply by dongle-the-gadget** [reply]
I know this is available for the Accent rule, not sure about Blur Behind though.
